src/vue/mixins/verification-form.mixin.js

```diff
@@ -0,0 +1,164 @@
+import FormMixin from '@/vue/mixins/form.mixin'
+
+import { vuexTypes } from '@/vuex'
+import { mapGetters, mapActions } from 'vuex'
+
+import { Sdk } from '@/sdk'
+import { base, ACCOUNT_TYPES } from '@tokend/js-sdk'
+
+import { DOCUMENT_TYPES } from '@/js/const/document-types.const'
+import { REQUEST_STATES_STR } from '@/js/const/request-states.const'
+import { BLOB_TYPES } from '@/js/const/blob-types.const'
+
+import { DocumentContainer } from '@/js/helpers/DocumentContainer'
+import { ErrorHandler } from '@/js/helpers/error-handler'
+
+const KYC_LEVEL_TO_SET = 0
+const KYC_CREATION_REQUEST_ID = '0'
+
+export default {
+  mixins: [FormMixin],
+  computed: {
+    ...mapGetters({
+      kycLatestData: vuexTypes.kycLatestData,
+      kycState: vuexTypes.kycState,
+      kycRequestId: vuexTypes.kycRequestId,
+      account: vuexTypes.account,
+    }),
+  },
+  async created () {
```

Comment:
I don't think it's a good idea for mixins to have hooks. You implicitly injecting the behavior, this have following effects:
 * non-obvious logic during debug processes
 * when you'll want to add some logic before the `await loadKyc()` part (and it will be specific for the component where you're injecting your mixin), you'll face some problems

I'd suggest having methods we can call in the component's hooks.

Reply:
Resolved, moved